KRuntime.sln

```diff
@@ -79,6 +65,20 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Runtime
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "DependencyAnalyzer", "src\DependencyAnalyzer\DependencyAnalyzer.kproj", "{C70D1E4C-D41A-4302-98B0-194B92E49043}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "dotnet.core45.managed", "src\dotnet.core45.managed\dotnet.core45.managed.kproj", "{1DEBD970-0B84-41F7-B247-D4E853432432}"
```

Comment:
will we need a net45 -> net46 rename in the near future?  might be better to choose names w/o versions now